src/pages/studyView/addChartButton/genericAssaySelection/GenericAssaySelection.tsx

```diff
@@ -85,20 +118,30 @@ export default class GenericAssaySelection extends React.Component<
     }
 
     @computed
-    private get entityNotSelected() {
-        return _.isEmpty(this._selectedGenericAssayEntityIds);
+    private get buttonDisabled() {
+        return (
+            this.props.studyViewMode &&
+            _.isEmpty(this._selectedGenericAssayEntityIds)
+        );
     }
 
     @computed
     private get molecularProfileOptions() {
         if (this.props.molecularProfileOptions) {
-            return this.props.molecularProfileOptions.map(option => {
-                return {
-                    ...option,
-                    label: `${option.label} (${option.count} samples)`,
-                    profileName: option.label,
-                };
-            });
+            if (this.props.studyViewMode) {
```

Comment:
i don't love this studyViewMode thing.  Especially b/c this componont lives in studyView folder.  Is there a way we could keep this fork outside of the component?   note that it seems to consume ONLY from props, so why not just do it in parent or store?